workflow_tests/utils/utils.ts

```diff
@@ -1,9 +1,23 @@
-const yaml = require('yaml');
-const fs = require('fs');
-const path = require('path');
+import type {StepIdentifier} from '@kie/act-js/build/src/step-mocker/step-mocker.types';
+import fs from 'fs';
+import path from 'path';
+import yaml from 'yaml';
+import type {ExtendedAct} from './ExtendedAct';
+
+type EventOptions = {
+    action?: string;
+};
 
-function setUpActParams(act, event = null, eventOptions = null, secrets = null, githubToken = null, envVars = null, inputs = null) {
-    let updated_act = act;
+function setUpActParams(
+    act: ExtendedAct,
+    event: string | null = null,
+    eventOptions: EventOptions | null = null,
+    secrets: Record<string, string> | null = null,
+    githubToken: string | null = null,
+    envVars: Record<string, string> | null = null,
+    inputs: Record<string, string> | null = null,
```

Comment:
If we want to clean this up I'd go with named params - that way marking them as optional would make more sense. Right now there are a lot of cases where this function gets a null just to maintain the order of the params and I'd like to avoid changing there places where it is being referenced. Nonetheless, I would keep it for the other pull request, because there are a lot of things that require refactoring.